domains/khanhdinh.json

```diff
@@ -0,0 +1,10 @@
+{
+    "description": "Describe the use of this subdomain",

```

Comment:
optional but if can please provide a clear description or just remove this entirely